### PR TITLE
[IMP][14.0]account_loan: add loan lines menu item

### PR DESCRIPTION
**Pull request by odooNextev**

### CODE REVIEW

**Review by bosd — APPROVED** [review]
Functional review LGTM :+1:

**Review by francesco-ooops — APPROVED** [review]
Functional ok!

Added loan items menu with pivot and graph for loan analysis.

**Review by GSLabIt — APPROVED** [review]

**Comment by francesco-ooops** [comment]
@AaronHForgeFlow since I saw you recently merged the v16 migration, is this ok? we can then forward-port it.

**Comment by AaronHForgeFlow** [comment]
For me it is fine. WDYT  @etobella ?

**Review by etobella — COMMENTED** [review]
I like the idea. Can you do something like this https://github.com/odoo/odoo/blob/14.0/addons/account/views/account_move_views.xml#L201-L207 in order to improve even more the view :smile:

**Comment by francesco-ooops** [comment]
Hi @etobella , what is it that you suggest to change functionally? 

AFAICS it's not currently possible to edit loan items from loan form

**Comment by etobella** [comment]
No, no, I just wanted to add an access to the loan :smile:

**Comment by odooNextev** [comment]
Now, clicking on the  loan name it will open loan form view.

File: account_loan/views/account_loan_lines_view.xml [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+                <field name="interests_amount" />
+                <field name="long_term_pending_principal_amount" />
+                <field name="long_term_principal_amount" />
+                <field name="company_id" optional="hide" />
+                <field name="partner_id" optional="hide" />
+                <field name="journal_id" optional="hide" />
+                <field name="short_term_loan_account_id" optional="hide" />
+                <field name="long_term_loan_account_id" optional="hide" />
+                <field name="interest_expenses_account_id" optional="hide" />
+            </tree>
```
Comment (etobella) [inline account_loan/views/account_loan_lines_view.xml:27]:
```suggestion
                    <groupby name="loan_id">
                        <button name="edit" type="edit" icon="fa-edit" title="Edit"/>
                    </groupby>
            </tree>
```

File: account_loan/views/account_loan_lines_view.xml [hunk 2]
```diff
@@ -0,0 +1,61 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<odoo>
+    <record id="account_loan_lines_view" model="ir.ui.view">
+        <field name="name">account.loan.lines.view</field>
+        <field name="model">account.loan.line</field>
+        <field name="arch" type="xml">
+            <tree create="0" import="0" editable="bottom">
```
Comment (etobella) [inline account_loan/views/account_loan_lines_view.xml]: Do not add this. It might be a problem to edit the information

**Review by etobella — COMMENTED** [review]
I was not talking about this. I was looking for something like this:

This would add an edit button to access the loan when grouping by loan :smile:

File: account_loan/views/account_loan_lines_view.xml (hunk 1 above)
Comment (odooNextev) [reply account_loan/views/account_loan_lines_view.xml:27]: Thanks for the advices. Now it shown the edit button when grouping.

**Comment by etobella** [comment]
Thanks, as you are adding a menu, can you add a new parent menu to join them? Now it looks like this:

image

Sorry to ask so much, but I think we can improve it with just a small change.

**Review by etobella — APPROVED** [review]
/ocabot merge patch

Can you port it to 16? Module has not been migrated to 15

**Pull request merged**